src/DateInput/index.scss

```diff
@@ -1,5 +1,3 @@
-@import "flatpickr/dist/themes/airbnb.css";
```

Comment:
Moved to main block of third party and generated CSS imports in main SCSS entry.